package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "start": "yarn && react-native start",
     "run-android": "react-native run-android",
-    "run-ios": "cd ios && pod install && cd .. && react-native run-ios",
+    "run-ios": "yarn install:pod && react-native run-ios",
```

Comment:
are we assuming everyone will use yarn?

Reply:
sorry ... I meant that to be a comment, not request changes.  If realistically everyone must use yarn, then I think this is OK.  I'm just not sure.

Reply:
Oh I see. yes we enforce yarn for this repo (over npm). so they wont get far without it